wordle-web/src/components/LetterButton.vue

```diff
@@ -1,8 +1,5 @@
 <template>
-  <LetterBase
-    :char="props.letter.char"
-    :color="props.letter.color"
-  ></LetterBase>
+  <LetterBase :char="props.letter.char" :color="props.letter.color"></LetterBase>
```

Comment:
It seems like you have alot of unrelated changes in your assignment. If you want to make changes unrelated to the assignment, make them in the branch you are making your PR against, that way they don't appear as changes.